ambry-api/src/main/java/com.github.ambry/router/GetBlobOptions.java

```diff
@@ -14,46 +14,57 @@
 
 package com.github.ambry.router;
 
+import com.github.ambry.protocol.GetOptions;
+
+
 /**
  * Represents any options associated with a getBlob request when making a
  * {@link Router#getBlob(String, GetBlobOptions, Callback)} call.
  */
 public class GetBlobOptions {
```

Comment:
`GetBlobOptions` on the other makes sense as a plural as an instance can have several associated options related to the `GetBlob` operation.